clr-k8s-examples/7-rook/overlays/v1.1.0/patch_cephcluster.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: rook-ceph
   namespace: rook-ceph
 spec:
+  mon:
+    allowMultiplePerNode: true
   storage:
     directories:
-     - path: /var/lib/rook
\ No newline at end of file
+     - path: /var/lib/rook
```

Comment:
hehe, spurious editor injected extra line feeds? ;-)

Reply:
Bah! fixing it. :)